forge-gui/res/cardsfolder/n/night_terrors.txt

```diff
@@ -1,5 +1,6 @@
 Name:Night Terrors
 ManaCost:2 B
 Types:Sorcery
-A:SP$ ChangeZone | Cost$ 2 B | Origin$ Hand | Destination$ Exile | ValidTgts$ Player | DefinedPlayer$ Targeted | Chooser$ You | ChangeType$ Card.nonLand | ChangeNum$ 1 | IsCurse$ True | Mandatory$ True | SpellDescription$ Target player reveals their hand. You choose a nonland card from it. Exile that card.
+A:SP$ RevealHand | Defined$ Targeted | ValidTgts$ Player | SubAbility$ ExileCard | StackDescription$ SpellDescription | SpellDescription$ Target player reveals their hand.
+SVar:ExileCard:DB$ ChangeZone | Origin$ Hand | Destination$ Exile | DefinedPlayer$ Targeted | Chooser$ You | ChangeType$ Card.nonLand | ChangeNum$ 1 | IsCurse$ True | StackDescription$ {p:You} chooses a nonland card from it. Exile that card. | SpellDescription$ You choose a nonland card from it. Exile that card.
```

Comment:
This still needs Mandatory?

Reply:
i wonder if there is an Param for revealing hand in the ChangeZone Effect

Reply:
Isn't a zone still hidden if it has been revealed? So you aren't required to find in this case?

Reply:
This effect doesn't do any "searching" ⚠️